tests/furusiya/model/event/test_event_bus.py

```diff
@@ -0,0 +1,49 @@
+from unittest.mock import Mock
+
+import pytest
+
+from model.event.event_bus import EventBus
+
+
+class TestEventBus:
+    @pytest.fixture
+    def event_bus(self):
+        yield EventBus()
```

Comment:
I would stop writing fixtures for things that are trivial to construct, and just construct them inline in tests. It makes the test classes less noisy.